tool/lib/license_utils.dart

```diff
@@ -0,0 +1,378 @@
+// Copyright 2024 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:path/path.dart' as p;
+import 'package:yaml/yaml.dart';
+
+/// This class contains config file related business logic for
```

Comment:
at the top of this file, can you provide instructions / docs on how this dart script should be used? Will this be added as a `dt` command? Ran on each commit on the CI?